requirements.txt

```diff
@@ -1,3 +1,7 @@
 -r requirements_base.txt
 
-mysqlclient==2.2.0        # via -r requirements.in
+mysqlclient~=1.4.6
+
+docutils~=0.19
+setuptools~=68.2.2
+pandas~=2.1.1
```

Comment:
Please don't add pandas to the requirements file. We don't have a great solution in place for local dependency tracking yet. But, since pandas wouldn't be needed for the server code, we shouldn't have the server code install it (and the server installs everything from this file).

Also, it's a complicated process to get new libraries approved and included into the RDR codebase.

Reply:
is there a way to ignore the missing requirement for the tool then?